code/parse/sexp.cpp

```diff
@@ -1762,6 +1762,16 @@ int check_operator_argument_count(int count, int op)
 	return 1;
 }
 
+template <typename T>
+int count_named_items(const char *name, int num, const T* item)
```

Comment:
Since `item` is an array, please rename to `items` or similar.

For `num` I'd prefer `num_items` or `size`.

Also, I think it's more common (and more intuitive) to have the array size argument after the pointer/array itself.

Reply:
Also, the name `count_named_items()` is a bit unintuitive IMO. Maybe something like `count_items_matching_name()`, or maybe `count_matching_items()`/`count_occurrences()`  and rename the first argument to `name_to_match`.

Reply:
ok, messed around with this